app/views/catalog/_hero_image.html.erb

```diff
@@ -3,6 +3,7 @@
 <div class="carousel slide mt-3 mb-3" data-ride="carousel" data-interval="6000">
   <div class="carousel-caption">
     <h1 class="intro-hero-title">Emory Digital Collections</h1>
+    <h3 class="intro-hero-text">Access unique research materials</h3>
```

Comment:
We have a .header .-h3 class defined in Lux that mimics the size/weight of the pattern library h3 style: https://github.com/emory-libraries/dlp-lux/blob/master/app/assets/stylesheets/lux/_results_list.scss